model/registry/system.yaml

```diff
@@ -43,6 +43,7 @@ groups:
               stability: experimental
         brief: "The state of the CPU"
         stability: experimental
+        deprecated: 'Removed, report cpu state with `cpu.state` common attribute'
```

Comment:
deprecated attributes should be extracted into it's own file in `registry/deprecated`

Reply:
system and process already exist there.

Reply:
@ChrsMark I see this is fixed already. Resolving this.